conan/tools/files/files.py

```diff
@@ -9,16 +9,15 @@
 import sys
 from contextlib import contextmanager
 from fnmatch import fnmatch
-
-import six
 from urllib.parse import urlparse
 from urllib.request import url2pathname
 
+import six
```

Comment:
No six import should be necessary anymore! I thought we had gotten rid of this 😅